quickstart.rst

```diff
@@ -157,7 +157,7 @@ Services within a Flux session may be implemented by modules loaded in the ``flu
 
 .. code-block:: console
 
-  $ flux module list --rank=all
+  $ flux module list
   Module                   Size Digest  Idle  S   Nodeset Service
   job-ingest            1143808 AF0B59E    4  S     [0-2]
```

Comment:
The output of `flux module list` also needs to be updated. It no longer includes `Nodeset` since it only targets a single rank.

Reply:
Thanks for catching that @grondo! I went ahead and updated the output and force-pushed.